README.md

```diff
@@ -64,9 +64,9 @@ Give a ⭐️ if you like this project!
 
 ## Acknowledgments
 
+- Microverse
 - Hat tip to anyone whose code was used
-- Inspiration
-- etc
+- My supportive family 😊
 
 ## 📝 License
 
```

Comment:
- Please fix the MIT link for a professional README file

Reply:
@MahmoudBakr23 I added the MIT.md file, thanks for the recommendation